src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkWindows.cs

```diff
@@ -108,6 +108,10 @@ protected override IEnumerable<string> GetAllAvailableAndroidSdks ()
 			foreach (var basePath in paths)
 				if (Directory.Exists (basePath))
 					yield return basePath;
+
+			// check for environment variables last.
+			foreach (string dir in GetSdkFromEnvironmentVariables ())
+				yield return dir;
```

Comment:
Does this actually need to check "next to last"? Where it would check env vars before the hardcoded paths?

I'm thinking when `%localappdata%\Android\android-sdk` exists, and someone sets `%ANDROID_HOME%` it wouldn't work.

Reply:
I'm not sure, it wouldn't pick it up at all with the current code, so I wasn't sure if I should change the existing behaviour (e.g it would pick up and use `%localappdata%\Android\android-sdk` rather than `ANDROID_HOME`.

Reply:
I moved this up a bit after the registry checks as per the chat on the meeting yesterday